class.jetpack.php

```diff
@@ -1283,18 +1283,16 @@ public function check_twitter_tags() {
 	 * @return null
 	 */
 	public static function perform_security_reporting() {
-		$last_run = Jetpack_Options::get_option( 'last_security_report' );
+        $no_check_needed = get_site_transient( 'security_report_performed_recently' );
 
-		$fifteen_minutes_ago = time() - ( 15 * MINUTE_IN_SECONDS );
-
-		if( $last_run > $fifteen_minutes_ago ) {
+		if( isset( $no_check_needed ) && $no_check_needed ) {
```

Comment:
I think you want to use `! empty()` here, since `isset()` will return true no matter what in this case.  Might not even be necessary since `get_site_transient()` already returns false if empty or doesn't exist.